content/w/johnny-blade.md

```diff
@@ -9,17 +10,102 @@ country = ["PL"]
 1 = { path = "johnny-blade.jpg", caption = "Promo shot of Johnny Blade.", source = "Johnny Blade's official Facebook" }
 +++
 
+Johnny Blade, also known under numerous aliases, is a Polish wrestler best known for his time in [PpW Ewenement](@/o/ppw.md). He also made a few appearances in [Dream Factory Wrestling](@/o/dfw.md) and [Maniac Zone Wrestling](@/o/mzw.md).
+
+## Basic info
+
+* Years active: 2011-present
+* Gimmicks/nicknames:
+  - Johnny Blade {{ org_badge(orgs=['ppw', 'mzw']) }}
+  - Ultimo Combo {{ org_badge(orgs=['ppw']) }}
+  - Angel Guerrero {{ org_badge(orgs=['ppw']) }}
+  - Epic Angel Guerrero {{ org_badge(orgs=['ppw']) }}
+  - Czacha {{ org_badge(orgs=['ppw']) }}
+  - Black Orion {{ org_badge(orgs=['ppw', 'dfw']) }}
+  - Human {{ org_badge(orgs=['ppw']) }}
+  - Vins Michael $hot {{ org_badge(orgs=['ppw']) }}
+* Trained by:
+  - PpW Dojo
+  - [Ewenement Dojo](@/o/ewenement-dojo.md) (2024-)
+
+## Professional wrestling career
+
+### PpW Ewenement (2011-present)
+
+#### Backyard era
+
+Johnny Blade was one of the founding members of [PpW](@/o/ppw.md) back in 2010/2011, along with his fellow classmates. Since the creation of this backyard group, Johnny was one of the wrestlers who made numerous appearances at early PpW shows. Like his fellow wrestlers, he worked under multiple gimmicks and aliases, often wearing a mask.
+
+In his early days, his main persona was a masked luchador called Ultimo Combo. It was used from the earliest days of PpW, until about 2016. Occasionally, he also performed under a comedic gimmick of "the worst high-flyer in PpW" Angel Guerrero, as an unmasked authority persona Vins Michael $hot, and "Human" - a short-lived horror gimmick of a mutant.
+From 2016 to 2019 he also worked under his more tag-team oriented gimmick of [Automatico's](@/w/rob-scaffold.md) cousin: Ultimo Combo.
```

Comment:
UCombo is mentioned as his main persona in line 39 above, so at least this needs rewording.